vision_msgs_rviz_plugins/CMakeLists.txt

```diff
@@ -0,0 +1,153 @@
+cmake_minimum_required(VERSION 3.5)
+project(vision_msgs_rviz_plugins)
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(Qt5 REQUIRED COMPONENTS Widgets Core)
+find_package(yaml-cpp REQUIRED)
```

Comment:
again, I think this is a vendor

Reply:
Should I choose this implementation instead: https://github.com/ros2/yaml_cpp_vendor ?

Reply:
Yeah! See comment above